connector/rest/src/main/java/it/pagopa/selfcare/party/registry_proxy/connector/rest/client/GeoTaxonomiesRestClient.java

```diff
@@ -0,0 +1,20 @@
+package it.pagopa.selfcare.party.registry_proxy.connector.rest.client;
+
+import it.pagopa.selfcare.party.registry_proxy.connector.rest.model.geotaxonomy.GeographicTaxonomiesResponse;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
```

Comment:
In case you'll have to add other methods you may consider the opportunity to generate the rest client e.g. through https://github.com/swagger-api/swagger-codegen (it's ok as-is if you have only this method).